go-controller/pkg/util/ovs.go

```diff
@@ -395,19 +394,25 @@ func RunOVNSBAppCtl(args ...string) (string, string, error) {
 		runner.ovnRunDir + sbdbCtlSock,
 	}
 	cmdArgs = append(cmdArgs, args...)
-	stdout, stderr, err := runOVNretry(runner.appctlPath, nil, cmdArgs...)
+	stdout, stderr, err := runOVNretry(runner.ovnappctlPath, nil, cmdArgs...)
 	return strings.Trim(strings.TrimSpace(stdout.String()), "\""), stderr.String(), err
 }
 
 // RunOVNNorthAppCtl runs an 'ovs-appctl -t ovn-northd command'.
 func RunOVNNorthAppCtl(args ...string) (string, string, error) {
 	var cmdArgs []string
+
+	pid, err := ioutil.ReadFile(runner.ovnRunDir + "ovn-northd.pid")
+	if err != nil {
+		return "", "", fmt.Errorf("failed to run the command since failed to get ovn-northd's pid: %v", err)
+	}
+
 	cmdArgs = []string{
 		"-t",
-		runner.ovnRunDir + northdCtlSock,
+		runner.ovnRunDir + fmt.Sprintf("ovn-northd.%s.ctl", strings.Trim(string(pid), " \n")),
```

Comment:
@girishmg `strings.TrimSpace(string(pid))` which trims '\t', '\n', '\v', '\f', '\r', ' ', 0x85, and 0xA0

Reply:
@dcbw will do. thanks.